mission-control/docs/playbooks/concepts/templating.md

```diff
@@ -4,7 +4,7 @@
 
 > Simply a way to represent data in different forms.
 
-Templating allows your playbook actions to work in context of a config or a component. Some actions, like exec action, can be templated using [Go templates](https://golang.org/pkg/text/template/).
+Templating allows your playbook actions to work in context of a config, health check or a component.
```

Comment:
I think lets copy  this into the actions directly, so I can see whats available from the action without needing to refer to concepts. 

For the field definitions we need to copy them into a Reference or Types page  we can refer to the go docs.